README.md

```diff
@@ -5,6 +5,7 @@
 [![Build][travis-badge]][travis]
 [travis]: https://travis-ci.org/git-afsantos/jTuples
 [travis-badge]: https://api.travis-ci.org/git-afsantos/jTuples.svg
+[![Coverage Status](https://coveralls.io/repos/github/git-afsantos/jTuples/badge.svg?branch=master)](https://coveralls.io/github/git-afsantos/jTuples?branch=master)
```

Comment:
I think the badges look better in the same line, like in [here](https://github.com/google/guava#guava-google-core-libraries-for-java).